docs/upgrade/9_to_10/mysql5.sql

```diff
@@ -0,0 +1,2 @@
+-- Increase mime_type field size
+ALTER TABLE oc_assets_asset MODIFY COLUMN mime_type VARCHAR (255);
```

Comment:
If this is for `develop`, this should be the upgrade script from 10 to 11.
The old ones shouldn't exist, since you wouldn't run this multiple times.

Reply:
Hi @lkiesow, do you mean I should remove the "update" statement from the  "docs/upgrade/8_to_9/mysql5.sql" script?

Reply:
Yep, those need to go before we can merge this in.